src/toolbox.ts

```diff
@@ -1929,38 +1945,38 @@ export class KeypointSliderItem extends ToolboxItem {
             filter_value = Math.round(this.filter_value * 100);
         }
         // Store which annotations need to be redrawn
-        let annotations_ids_to_redraw_by_subtask: {[key: string]: string[]} = {}
+        const annotations_ids_to_redraw_by_subtask: { [key: string]: string[] } = {};
         // Initialize the object with the subtask keys
-        for (let subtask_key in ulabel.subtasks) {
-            annotations_ids_to_redraw_by_subtask[subtask_key] = []
+        for (const subtask_key in ulabel.subtasks) {
+            annotations_ids_to_redraw_by_subtask[subtask_key] = [];
         }
 
         // Get all point annotations
         const point_and_line_annotations = get_point_and_line_annotations(ulabel);
         for (const annotation of point_and_line_annotations[0]) {
             // Get the annotation's confidence as decimal between 0-1
-            let confidence: number = this.get_confidence(annotation)
+            let confidence: number = this.get_confidence(annotation);
 
             // filter_value will be a number between 0-100, so convert the confidence to a percentage as well
-            confidence = Math.round(confidence * 100)
+            confidence = Math.round(confidence * 100);
 
             // Compare the confidence value against the filter value
-            const should_deprecate: boolean = this.filter_function(confidence, filter_value)
+            const should_deprecate: boolean = this.filter_function(confidence, filter_value);
             // Check if an annotation should be deprecated or undeprecated, else do nothing
             if (
-                (should_deprecate && !annotation.deprecated) ||
-                (!should_deprecate && annotation.deprecated)
+                (should_deprecate && !annotation.deprecated)
+                || (!should_deprecate && annotation.deprecated)
```

Comment:
thoughts on this? iirc prettier had it the other way with the logical operator hanging on the first line, and i think i slightly prefer that

Reply:
I prefer it this way (same as the `.` before properties) but am not super opinionated on it. 
There's a few rules that affect this:
- [indent-binary-ops](https://eslint.style/rules/default/indent-binary-ops)
- [operator-linebreak](https://eslint.style/rules/default/operator-linebreak)
- [multiline-ternary](https://eslint.style/rules/default/multiline-ternary) (ternary only)
- [no-mixed-operators](https://eslint.style/rules/default/no-mixed-operators) (I doubt we'd enable this)